policy/approval/approve_test.go

```diff
@@ -188,6 +194,23 @@ func TestIsApproved(t *testing.T) {
 		assertPending(t, prctx, r, "0/1 required approvals. Ignored 7 approvals from disqualified users")
 	})
 
+	t.Run("singleApprovalError", func(t *testing.T) {
+		prctx := basePullContext()
+		// The converse of the check in `noApprovalRequired`. If there are
+		// approvers required, we do expect to call `Comments()`, and therefore
+		// this error should be returned to show that it was called.
+		prctx.CommentsError = errors.New("Comments() was called")
```

Comment:
I don't mind having a specific test for this, but I think the fact that rules with required approvals call `Comments()` should be covered by the other approval tests, right?

Reply:
That makes sense. I added it initially while developing so it'd go red if I made a mistake. It doesn't need to be committed. caa985cfeabaa10247c5c34f8e2f38d3a4fcccc4